codalab/lib/bundle_cli.py

```diff
@@ -1612,6 +1608,12 @@ def derive_bundle(self, bundle_type, command, targets, metadata):
                 help='If a bundle with the same command and dependencies already exists, return it instead of creating a new one.',
                 action='store_true',
             ),
+            Commands.Argument(
+                '-i',
+                '--interactive',
+                help='Beta feature - Start an interactive session to construct your run command.',
```

Comment:
```suggestion
                help='Start an interactive session to construct your run command.',
```

I think we should remove this. We have "beta feature" in a few other commands, which are really already released and supported by us.

Reply:
Calling the other features "beta" might be incorrect, but this is really still a beta feature.